tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -295,10 +282,158 @@ def finalizer():
 
         self.dynamic_pvc_base(interface_type, reclaim_policy)
 
-    def rwx_dynamic_pvc(self, setup_base):
-        logger.info('RWX Test')
+    def test_rwx_dynamic_pvc(self, setup_base):
+        """
+        RWX Dynamic PVC creation tests with Reclaim policy set to Delete/Retain
+        """
+        logger.info(f"CephFS RWX test")
+        logger.info(f"Creating two pods using same PVC {self.pvc_obj.name}")
```

Comment:
lines 290-316 are similar to 176-196.
Please create a common function for them.
This function should be a fixture for the test, so the pods created there will be deleted in the teardown and not in the test in lines 218, 243, 388-392.

Reply:
@ebenahar moved creation of first pod in the common function "dynamic_pvc_base" since the basic setup part for all test requires atleast the first pod to be in Running state.

The check for second pod is different for both RWO and RWX, hence didnt move them to the common method.

Please review.

Reply:
ok, thanks